components/binance/browser/binance_json_parser.cc

```diff
@@ -373,10 +373,12 @@ bool BinanceJSONParser::RevokeTokenFromJSON(
     return false;
   }
 
-  if (!response_dict->GetBoolean("success", success_status)) {
+  absl::optional<bool> success_optional =
+      response_dict->FindBoolPath("success");
```

Comment:
`FindBoolKey` should be used. `*Path` is for `some.sub.key` lookups.

Reply:
Fixed in 2713c0a45d9d84f7cb4a310c982e48a7064bd900